guides-markdown/multisig-wallet.md

```diff
@@ -0,0 +1,370 @@
+
+```post-author
+Joe Miyamoto <joemphilips@gmail.com>
+```
+
+```post-description
+How to use bcoin with a multisig wallet and account. For both as a hot-wallet and as a watch-only-wallet.
+```
+
+## How it works
+
+Bitcoin multisig account is an account which holds more than 1 public key,
+and generates P2SH or P2WSH address for those public keys.
+When you want to use those funds,
+you must create a multisig transaction which has been signed by private keys corresponding to those public keys.
+See [our multisig transaction guide](http://bcoin.io/guides/multisig-tx.html) for more detail about the multisig transaction itself.
+
+In this guide, we take it a little step further and see a whole lifecycle of the multisig account in a more practical way.
+
+## Usecases
+
+### 1. Personal wallet which is secure against key compromise
+
+Usually in a cryptocurrency world, when you lose your master key and it's seed for the wallet there is no way to recover your funds.
+But by splitting your secret keys into ...
+
+1. your own laptop.
+2. hardware wallet you own.
+3. a paper wallet in your personal vault
+
+...your funds will be safe unless more than one key is compromised at the same time.
+In this case, you may use 1. and 2. for usual spending,
+and only need 3 if you lose your secret keys 1 and 2.
+
+### 2. Wallet held by more than one person
+
+Sometimes it is desireable to distribute the control of funds among several people.
+For example, funds for a corporation or for a crowdfunding project.
+We can mitigate the risk of fund loss by utilizing a multisig in cases where single person with control:
+
+1. Is no longer in charge
+2. Disappears suddenly
+3. Loses their secret-key
+4. Tries to steal all funds for their own personal use
+
+## Walk-through
+
+ This tutorial includes the following steps
+ 1. setting up a bcoin wallet server with https
+ 2. setting up a hot wallet on the server
+ 3. setting up a watch-only multisig wallet on your server
+ 4. receive incoming information from the wallet server
+ 5. spend funds from the wallet
+ 6. recovering the wallet
+
+### 1. Setting up bcoin wallet server with https.
+
+Q. Why using https(ssl/tls) ?
+
+A. There are two reason to use https.
+
+1. For better privacy and security ... since all commands to a server is encrypted, no one is able to eavesdrop the communication. This is a same motivation behind [bip150](https://github.com/bitcoin/bips/blob/master/bip-0150.mediawiki) and [bip151](https://github.com/bitcoin/bips/blob/master/bip-0151.mediawiki)
+2. To collaborate with others easily ... in later secrion, we are showing how to use watch-only wallet with more than one people. In those situation, some might not be familiar with accessing the server with IP address, or want to access from browser, or don't want to bother himself remember(and track) IP address for the server.
+
+We assume you already have gotten your domain name somehow,
+and that is `bcoin.multisigtest.com`.
```

Comment:
Do you think it would be easier to create a self signed cert for `localhost` so that users can more easily follow along?